spec/undercover_spec.rb

```diff
@@ -1,46 +1,40 @@
-require File.expand_path("../spec_helper", __FILE__)
+# frozen_string_literal: true
+
+require File.expand_path('spec_helper', __dir__)
 
 module Danger
   describe Danger::DangerUndercover do
-    it "should be a plugin" do
+    it 'should be a plugin' do
```

Comment:
Prefers the imperative:

```suggestion
    it 'is a Danger plugin' do
```

> From Compass:
> ![image](https://user-images.githubusercontent.com/696529/95814164-ecb22580-0d43-11eb-9568-5bb2c17fc916.png)